examples/with-nextjs-and-clerk-auth/src/monite-theme.d.ts

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+
+import '@mui/material/styles';
+import '@mui/x-data-grid/themeAugmentation';
+
+declare module '@mui/material/styles' {
+  interface TypeBackground {
+    default: string;
+    paper: string;
+    menu: string;
+    highlight: string;
+  }
+
+  interface Palette {
+    neutral: {
+      '10': string;
+      '30': string;
+      '50': string;
+      '70': string;
+      '80': string;
+      '90': string;
+      '95': string;
+      '100': string;
+    };
+  }
+
+  interface PaletteOptions {
+    neutral?: {
+      '10': string;
+      '30': string;
+      '50': string;
+      '70': string;
+      '80': string;
+      '90': string;
+      '95': string;
+      '100': string;
+    };
+  }
+
+  interface TypographyVariants {
+    label1: React.CSSProperties;
+    label2: React.CSSProperties;
+    label3: React.CSSProperties;
```

Comment:
As I might have noticed, `label1` is not used anywhere for styling.
`label2` is used in three places, and in only two of the cases in different contexts.
`label3` is used 1 time.
Furthermore, I don't see a significant difference in styles in these variants, which contradicts the idea of style tokens.
I'd recommend focusing on tweaking the sentient options, and where standard options aren't enough - styling components where required.